templates/contrib/jdbc/README.md

```diff
@@ -78,10 +78,10 @@ The namespace is the project that you are working on OpenShift, or, you can also
 
 ## Building a extension image
 
-All you need to do is install cekit2 and execute the `make build` command specifying the image you want to build, i.e.:
+All you need to do is install cekit2 and execute the `make` command specifying the image you want to build, i.e.:
```

Comment:
I think there should be also mentioned Cekit 3.2. Same as my comments in #443

Reply:
answered there.